ups/rubintv_production.cfg

```diff
@@ -0,0 +1,20 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+# Dependencies that provide header files and or libraries should be included here.
+# Pure-Python dependencies do not need to be included.
+# Packages that use pybind11 or boost_tests should declare them as build dependencies.
+# Otherwise, the rules for which packages to list here are the same as those for
+# table files.
+dependencies = {
+}
+
+# For packages that build a C++ library and a Python module, the below should be sufficient.
+# Pure-Python packages should set headers=[], libs=[] (not libs=None). and hasSwigFiles=False.
+# For more information, see the sconsUtils Doxygen documentation.
+config = lsst.sconsUtils.Configuration(
+    __file__,
+    headers=[],
+    hasDoxygenInclude=False,
+)
```

Comment:
This file should be unnecessary for a pure-Python package.

Reply:
As discussed OOB, this does seem to be necessary 😕